doc/release-notes-18570.md

```diff
@@ -0,0 +1,2 @@
+- The `getbalances` RPC now returns `lastprocessedblock` JSON object which contains the wallet's last processed block
```

Comment:
nit: would add the undefined article "a" in front of `lastprocessedblock` here
```suggestion
- The `getbalances` RPC now returns a `lastprocessedblock` JSON object which contains the wallet's last processed block
```

Reply:
Thanks!